1-mandatory/2-people-i-know.js

```diff
@@ -393,15 +395,22 @@ You can see who people's colleagues are by seeing the "colleagues" array in each
 This time, I only want the full names ("<firstname> <lastname>") of my friends who are colleagues of hers.
 */
 
-let friendsWhoAreColleaguesOfStacie = [];
+//Happy to use chaining :)
+let friendsWhoAreColleaguesOfStacie = friends
+  .filter((el) => el.colleagues.find((el) => el.name == "Stacie Villarreal"))
+  .map((el) => el.name.first + " " + el.name.last);
+
 /*
 5) Find "Multi-tasking" colleagues
 Next, I want you to find all of the colleagues of my friends who are good at "Multi-tasking"
 You can tell if they are good at "Multi-tasking" because they will have it listed in their skills
 This time, I only want the full names of the people who can multitask
 */
 
-let colleaguesWhoCanMultitask = [];
+let colleaguesWhoCanMultitask = friends
+  .flatMap((el) => el.colleagues)
+  .filter((colleague) => colleague.skills.includes("Multi-tasking"))
+  .map((colleague) => colleague.name);
```

Comment:
This is a really excellent solution to this. The use of `.flatMap()` here is great.